### DISCUSSION TITLE
모우다 데모데이 전 버그 관리

### DISCUSSION THREAD

**Discussion by Mingyum-Kim** [woowacourse-teams/2024-mouda#657] [Announcements]
모우다 DEV 서버에서 발생하는 문제를 담당자 멘션과 함께 공유한다. 담당자가 이슈를 파서 문제를 해결하고 댓글에 해결 과정 (혹은 PR)을 작성한다.

우측 Notification을 설정해주세요. 메일로 알림이 가며 주기적으로 확인합시다.

![image](https://github.com/user-attachments/assets/8e9e9cb3-1390-4c14-9f8d-bfd0421c7c4b)

**Comment by Mingyum-Kim** [comment]
사용자 전환 시 예외 시나리오를 고려해서 사용자 친화적으로 변경하기
예외 상황을 사용자가 인지할 수 있도록 하기

@Mingyum-Kim @hoyeonyy  @jaeml06

**Comment by Mingyum-Kim** [comment]
이전에 애플 로그인한 사용자의 정보를 DB에서 자체로 삭제했을 때
애플에서 `user`를 보내지 않으면 데이터베이스에 새로 저장하지 않는다.
데이터베이스에 사용자 정보가 없으면 회원가입한 적이 없는 사용자이므로 로그인에 실패한다.

@Mingyum-Kim

**Comment by Mingyum-Kim** [comment]
프로필 사진 변경이 간헐적으로 안됨
모바일에서는 아예 안되는 것 같음
@Mingyum-Kim  @jaeml06 @hoyeonyy

**Comment by Mingyum-Kim** [comment]
버그는 아니지만 해야할 것
- 리프레시 토큰 도입
- 마이페이지 UI 변경
- 회원탈퇴 기능 테스트

@Mingyum-Kim @hoyeonyy @jaeml06

**Comment by Mingyum-Kim** [comment]
다락방을 탈퇴하는 기능도 필요해보입니다.
아래 사진의 `로그아웃` 위치에 다락방 나가기를 두는 게 어떨까요?
![image](https://github.com/user-attachments/assets/2c940787-303b-4877-b59e-c9d5c3b51d81)

**Comment by hoyeonyy** [comment]
현재 모임의 댓글에는 프사가 안뚫려있음
